robottelo/constants.py

```diff
@@ -1224,3 +1117,414 @@
     'Installable',
     '7/27/16'
 ]
+
+# To create custom role for bugzilla's
+# Append bugzilla id to dict, also see utils.get_perms_by_bz()
+PERMISSIONS_WITH_BZ = {
+    None: [
+        {'name': 'access_dashboard', 'bz': [1306359]},
+        {'name': 'access_settings', 'bz': [1306359]},
+        {'name': 'app_root', 'bz': []},
```

Comment:
`dict.get()` also accepts optional argument with default value, so you can use it like `dict.get('key', [])` and you won't be hit by None, don't have to store empty dicts and no need to import any custom dict implementation.